internal/cli/ops_manager_alerts_global_list.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	om "github.com/mongodb/go-client-mongodb-ops-manager/opsmngr"
+	"github.com/mongodb/mongocli/internal/flags"
+	"github.com/mongodb/mongocli/internal/json"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type opsManagerAlertsGlobalListOpts struct {
+	store  store.GlobalAlertLister
+	status string
+}
+
+func (opts *opsManagerAlertsGlobalListOpts) init() error {
+	var err error
+	opts.store, err = store.New()
+	return err
+}
+
+func (opts *opsManagerAlertsGlobalListOpts) Run() error {
+	alertOpts := om.AlertsListOptions{
```

Comment:
[q] what about pagination? 
[nit] you could also do `&om.AlertsListOptions{` here and not do the `&` on line 42